packages/gestalt/src/Masonry/defaultTwoColumnModuleLayout.test.js

```diff
@@ -4,6 +4,212 @@ import HeightsStore from './HeightsStore';
+      measurementStore.set(item, item.height);
+    });
+    const layout = defaultTwoColumnModuleLayout({
+      measurementCache: measurementStore,
+      positionCache,
+      justify: 'start',
+      rawItemCount: items.length,
+      width: 501,
+    });
+
+    expect(layout(items)).toEqual([
+      { top: 0, height: 100, left: 7, width: 236 },
+      { top: 0, height: 120, left: 257, width: 236 },
+      { top: 114, height: 80, left: 7, width: 236 },
+      { top: 134, height: 100, left: 257, width: 236 },
+    ]);
+  });
+
+  test('only centers when theres extra space', () => {
+    const measurementStore = new MeasurementStore<{ ... }, number>();
+    const positionCache = new MeasurementStore<{ ... }, Position>();
+    const items: $ReadOnlyArray<Item> = [
+      { 'name': 'Pin 0', 'height': 100 },
+      { 'name': 'Pin 1', 'height': 120 },
+      { 'name': 'Pin 2', 'height': 80 },
+      { 'name': 'Pin 3', 'height': 100 },
+    ];
+    items.forEach((item) => {
+      measurementStore.set(item, item.height);
+    });
+    const layout = defaultTwoColumnModuleLayout({
+      measurementCache: measurementStore,
+      positionCache,
+      justify: 'start',
+      rawItemCount: items.length,
+      width: 200,
+    });
+
+    expect(layout(items)).toEqual([
+      { top: 0, height: 100, left: 0, width: 236 },
+      { top: 0, height: 120, left: 250, width: 236 },
+      { top: 114, height: 80, left: 0, width: 236 },
+      { top: 134, height: 100, left: 250, width: 236 },
+    ]);
+  });
+
+  test('justify', () => {
+    const measurementStore = new MeasurementStore<{ ... }, number>();
+    const positionCache = new MeasurementStore<{ ... }, Position>();
+    const items: $ReadOnlyArray<Item> = [
+      { 'name': 'Pin 0', 'height': 100 },
+      { 'name': 'Pin 1', 'height': 120 },
+      { 'name': 'Pin 2', 'height': 80 },
+      { 'name': 'Pin 3', 'height': 100 },
+    ];
+    items.forEach((item) => {
+      measurementStore.set(item, item.height);
+    });
+
+    const makeLayout = (justify: 'center' | 'start') =>
+      defaultTwoColumnModuleLayout({
+        measurementCache: measurementStore,
+        positionCache,
+        columnWidth: 100,
+        gutter: 0,
+        justify,
+        width: 1000,
+        rawItemCount: items.length,
+      })(items);
+
+    const justifyStart = makeLayout('start');
+    positionCache.reset();
+    const justifyCenter = makeLayout('center');
+
+    expect(justifyStart).toEqual([
+      { top: 0, left: 0, width: 100, height: 100 },
+      { top: 0, left: 100, width: 100, height: 120 },
+      { top: 0, left: 200, width: 100, height: 80 },
+      { top: 0, left: 300, width: 100, height: 100 },
+    ]);
+
+    expect(justifyCenter).toEqual([
+      { top: 0, left: 300, width: 100, height: 100 },
+      { top: 0, left: 400, width: 100, height: 120 },
+      { top: 0, left: 500, width: 100, height: 80 },
+      { top: 0, left: 600, width: 100, height: 100 },
+    ]);
+  });
+});
+
```

Comment:
there was a twoCol test case here previously i think. did you want to add it back in?